ViewGenerator/tools/webpack/Plugins/CommonConfiguration.ts

```diff
@@ -130,11 +130,12 @@ let getCommonConfiguration = (libraryName: string, useCache: boolean, assemblyNa
             }),
         ]
     };
-    
+
     if (useCache) {
         Configuration.cache ={
             type: 'filesystem',
             allowCollectingMemory: true,
+            name: 'viewsCache'
```

Comment:
We cannot do this, as this file is shared between views and view plugins. We can follow the approach for cache name I did in https://github.com/OutSystems/ReactView/pull/154